File: src/Uri/_2_August/URI_1075_Remaning2.java
package Uri._2_August;

import java.util.Scanner;

public class URI_1075_Remaning2 {
    public static void main(String[] args) {
        int n;
        Scanner sc = new Scanner(System.in);
        n = sc.nextInt();

        for (int i = 1; i < 10000; i++) {
            if (i % n == 2) {
                System.out.println(i);
            }

        }

    }
}

File: src/Uri/_2_August/URI_1075_Remaning2.kt
package Uri._2_August

import java.util.*

fun main(args:Array<String>){
    var n:Int;
    var sc = Scanner(System.`in`);
    n = sc.nextInt();

    for (i in 1..10000){
        if (i % n == 2){
            println(i)
        }
    }
}